kubernetes/charts/oasis-platform/templates/oasis_worker_controller.yaml

```diff
@@ -50,12 +50,15 @@ spec:
       annotations:
         checksum/{{ .Values.oasisServer.name }}: {{ toJson .Values.oasisWebsocket | sha256sum }}
     spec:
+      #automountServiceAccountToken: false
```

Comment:
https://github.com/OasisLMF/OasisPlatform/blob/534c45ddb870818f00f514932d32cfeb155fe645/kubernetes/charts/oasis-platform/templates/oasis.yaml#L252-L273

`oasis-task-controller` is also missing its ReadOnly mount, but that will also need a Read/Write mount to the `/tmp` directory, otherwise it will also crash, like with celery beat.